hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/ReadableContractStateStore.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+ * Copyright (C) 2020-2024 Hedera Hashgraph, LLC
```

Comment:
Why are you back-dating the code?  That seems like it might be very much incorrect.

Reply:
I will change it back to 2023. I thought we want to be aligned with all the files

Reply:
I think we need to be "correct" on these, as it's a formal notice, not code.